sig-network/charter.md

```diff
@@ -34,16 +34,16 @@ The following topics fall under ownership of this SIG:
 - Network Policy
   - APIs for defining network policies (e.g. [NetworkPolicy], [AdminNetworkPolicy], [BaselineAdminNetworkPolicy])
 - Cluster DNS.
-- Integration points with networking implementations (e.g. CNI integration).
-- Container runtime interface (CRI) (With [sig-node]).
+- Integration points with networking implementations (e.g. [Container Network Interface (CNI)][CNI] integration).
```

Comment:
drop "integration" from the end of the sentence since it's already at the start?

Reply:
:+1: